docs/themes/legacy.rst

```diff
@@ -0,0 +1,91 @@
+Legacy Builder
+==========================================================
+
+.. Warning:: The legacy builder is still available in Mautic 4 but will be removed in Mautic 5. Refer to the `builder documentation <https://docs.mautic.org/en/builders>`__ for more information.
+
+Slots
+-----
+
+Slot Definition
+^^^^^^^^^^^^^^^
+
+The slot can be defined by a single HTML attribute ```data-slot="{slot type here}"````. For example, the text slot can be defined even with the demo content.
+
+When the theme is opened in the builder, the div with attribute ```data-slot="text"``` will make the text inside the div editable within the inline Froala editor.
+
+.. code-block:: html
+
+    <div data-slot="text">
+        <a>@JaneDoe</a> has invited you to join Awesome inc!
+    </div>
+
+The slot types currently built:
+
+Image
+"""""
+Inserts a single image into the div. User can click on it and edit it with options which provides Froala editor (link, change image source, alt text, ...)
+
+Button
+""""""
+Inserts a HTML button. User can define text, URL as well as padding, size and position.
+
+Text
+""""
+Inserts a new text slot which you can edit with a HTML editor, so you can insert even media like images and videos in it.
+
+Separator
+"""""""""
+Inserts a horizontal line to separate content.
+
+Slot Containers
+^^^^^^^^^^^^^^^
+As stated before, users can drag & drop the new slots into the theme. So as a theme developer, you have to define where the user can drop the slots. You can do it again with a single HTML attribute ``data-slot-container="1"``.
+
+.. code-block:: html
+
+    <div data-slot-container="1">
+        <div data-slot="text">
+            <a>@JaneDoe</a> has invited you to join Awesome inc!
+        </div>
+    </div>
+
+This way the builder will let users drop the new slots into this container. In the example above there is already one predefined slot which user can move to another container, remove or edit.
+
+This functionality will provide you with lots of creative freedom for designing and developing your own unique email and landing pages. Have a unique design? Share it with the community! We would love to see how you’re using Mautic to engage your audience.
```

Comment:
```suggestion
This functionality will provide you with lots of creative freedom for designing and developing your own unique Email and Landing Pages. Have a unique design? Share it with the community on the forums! We would love to see how you’re using Mautic to engage your audience.
```
When the external links are set up, please link to the forums at https://forum.mautic.org